docs/tutorials/python-development-environment.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Setting up a Python development environment
+subject: Software
+---
+By Anton Vanhoucke
```

Comment:
Because the tutorials are more "community content" than "personal content being shared", I think that the name should be more subtle than on the other pages (and we don't want to take up space on tutorials for the sidebar). I'm imagining a subtitle-like area below the title. Anyway, remind me in a week and I can try it out :wink: